examples/Raytracer_Benchmark/RaytraceBenchmark.cu

```diff
@@ -358,3 +367,40 @@ int RaytraceBenchmarkGPU(cuda::RaytracerData_t *rtdata, bool use_tiles, int bloc
   checkCudaErrors(cudaFree(output_buffer));
   return 0;
 }
+*/
+
+void cudaWorld(cuda::RaytracerData_t *rtdata) {
```

Comment:
The method name is not reflecting what it does - it is the implementation of specialization of  InitRTdata<GPU> - see comment in RaytraceBenchmark.hpp